legal-api/tests/unit/models/test_legal_entity.py

```diff
@@ -952,9 +921,10 @@ def test_alternate_names(session, test_name, legal_entities_info, alternate_name
                     start_date=start_date,
                     legal_entity_id=le.id,
                 )
-                le._alternate_names.append(alternate_name)
+                le.alternate_names.append(alternate_name)
 
         session.flush()
-        assert le.alternate_names == expected_alternate_names
+        for expected_alternate_name in expected_alternate_names:
+            assert any(l.json(slim=True)["alternateNames"][0] == expected_alternate_name for l in le.alternate_names)
```

Comment:
Cannot compare array of objects of `AlternateName` to the `expected_alternate_names`, so instead use `AlternateName.json`.